client.go

```diff
@@ -33,7 +33,10 @@ func NewClient(ifi *net.Interface) (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	return NewClientPacketConn(ifi, p)
+}
 
+func NewClientPacketConn(ifi *net.Interface, p net.PacketConn) (*Client, error) {
```

Comment:
Go style suggests that all top-level, exported names should have doc comments.

Reference: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments.

Also, Go lint would say:

```
exported function NewClientPacketConn should have comment or be unexported
```